### DISCUSSION TITLE
Batch set opacity of multiple layers at once

### DISCUSSION THREAD

**Discussion by hsandt** [Orama-Interactive/Pixelorama#992] [Ideas]
Currently, you can select multiple layers with Shift or Ctrl+click, but the shown opacity is only the one of the last selected layer, and changing it only edits the opacity or that layer.

It would be nice to allow batch editing the opacity of all selected layers at once.

**Comment by OverloadedOrama** [comment]
Implemented in cd2787b37306adaae34719e82523d20f33458b54 and it will be a part of update 1.0.
